.github/workflows/linters.yml

```diff
@@ -0,0 +1,20 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you would make the branch name more descriptive such as **feature/add-morse-decoder** instead of **feature** 👍 because your branch name should tell us what you are doing in this branch and also to make your pull request descriptive 😊